server/status/runtime.go

```diff
@@ -170,6 +179,18 @@ func (rsr *RuntimeStatSampler) SampleEnvironment() {
 		log.Errorf(context.TODO(), "unable to get cpu usage: %v", err)
 	}
 
+	fds := gosigar.ProcFDUsage{}
+	if err := fds.Get(pid); err != nil {
```

Comment:
`ProcFDUsage.Get` reads all of the entries in the `/proc/<pid>/fd` directory. That's a bit silly when `FDUsage.Get` reads either a single file (linux) or performs a syscall (freebsd) to get the number of open files. I wonder why the discrepancy.